packages/jellyfish-transaction/src/script/control.ts

```diff
@@ -8,3 +8,57 @@ export class OP_RETURN extends StaticCode {
     super(0x6a, 'OP_RETURN')
   }
 }
+
+export class OP_NOP extends StaticCode {
+  constructor () {
+    super(0x61, 'OP_NOP')
+  }
+}
```

Comment:
Add description for all, similar to how @jingyi2811 does it

Reply:
I have added the descriptions. Let me know if i got it correctly.